components/mongodb/mongodb.app.mjs

```diff
@@ -1,11 +1,234 @@
+import mongoose from "mongoose";
+
 export default {
   type: "app",
   app: "mongodb",
-  propDefinitions: {},
+  propDefinitions: {
+    collection: {
+      label: "Collection",
+      type: "string",
+      description: "Collection where the new document will be inserted.",
+      async options() {
+        return this.getCollections();
+      },
+    },
+    document: {
+      label: "Document",
+      type: "string",
+      description: "The document ID",
+    },
```

Comment:
What do you think about having this prop use async options to list all of the documents in a collection?

Reply:
I thought about it, but I don't think it is a good idea because we don't know what we should show as a description, since the data inside the same schema can change, we can not have sure if the description field of a document would be `name`, `description`, etc. Even the `_id` field can be changed. What do you think @dannyroosevelt?

Reply:
is any part of the response consistent? async options would definitely be ideal if possible

Reply:
The `_id` is the most consistent item, but even this field can be changed. We could list the `_id`, but sounds strange for me to list the `_id` to choose an `_id`. And the `_id` is something like: `61f082d577cedeacaa0eec45`. Looks strange for me to have a list of this.

Reply:
Yeah, I can try something like this :)